### ISSUE TITLE
Medic Ubercharge meter shows double %

### ISSUE THREAD

**Issue by jchristgit** [KinevoNetwork/HudasIscariote#1]
Hey, thank you for making this awesome hud! I play Medic and I really like the ubercharge bar on the bottom. I have an issue though, namely the `%` for uber shows up twice. ![image](https://cdn.discordapp.com/attachments/357529809959780362/379678274474999808/unknown.png)
When at 100%, it shows `10..[%]`. Is there a way to fix this? I'm playing on macOS with a screen resolution of 1440 x 900. Thanks in advance.

**Comment by DrFallen** [comment]
Hi there, I don't officially support Mac I'm afraid as I have no means to test it. You can probably trying playing around with the HudMedicCharge.res I can't remember the specific part of it.

**Comment by jchristgit** [comment]
Thanks for telling me about `HudMedicCharge.res`. I did some digging around today, and this is what I found:
- The `labelText` for `ChargeLabel` (which is the ubercharge text) is set to `#TF_UberchargeMinHUD`
- looked through TF2Classic's source code, [found out what it's set to](https://github.com/danielmm8888/TF2Classic/blob/d070129a436a8a070659f0267f6e63564a519a47/game/tf2classic/resource/tf2classic_english.txt#L1159), namely `%charge%%%`
- did some more testing around, and - voila!
 ![it works, nice](https://cdn.discordapp.com/attachments/357529809959780362/380727468237127702/unknown.png)
The fix was to use `"labelText"		"%charge%"`, so the full `ChargeLabel` settings in `HudMedicCharge.res` are the following now:
```json
	"ChargeLabel"
	{
		"ControlName"	"CExLabel"
		"fieldName"		"ChargeLabel"
		"xpos"			"c56"
		"ypos"			"15"
		"zpos"			"3"
		"wide"			"65"
		"tall"			"29"
		"autoResize"	"1"
		"pinCorner"		"2"
		"visible"		"1"
		"enabled"		"1"
		"tabPosition"	"0"
		"labelText"		"%charge%"
		"textAlignment"	"center"
		"dulltext"		"0"
		"brighttext"	"0"
		"fgcolor"		"255 85 0 255"
		"font"			"Uber30"
	}
```
I also tested this with the Vaccinator and it worked as expected. Thanks for your help!
